protocol/v2/message/msg.go

```diff
@@ -31,20 +32,19 @@ func MsgTypeToString(mt spectypes.MsgType) string {
 }
 
 // BeaconRoleFromString returns BeaconRole from string
-func BeaconRoleFromString(s string) spectypes.BeaconRole {
+func BeaconRoleFromString(s string) (spectypes.BeaconRole, error) {
```

Comment:
why we need error here? let's return unknown role instead

Reply:
@nivBlox 
We need it to avoid panic if role in request is malformed. We don't have a `BeaconRole` for an unknown role.

Reply:
so we can return unknown role and the one that used this func should handle this case

Reply:
in that case let's keep it with error